modules/programs/buf.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.buf;
+in {
+  meta.maintainers = [ maintainers.lucperkins ];
+
+  options.programs.buf = {
+    enable = mkEnableOption "Buf: a CLI tool for working with Protocol Buffers";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.buf;
+      defaultText = literalExpression "pkgs.buf";
+      description = "Package providing <command>buf</command>.";
+    };
+
+    enableBashIntegration = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to load Bash autocomplete scripts for <command>buf</command>.
+      '';
+    };
+
+    enableFishIntegration = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to load Fish autocomplete scripts for <command>buf</command>.
+      '';
+    };
+
+    enableZshIntegration = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to load zsh autocomplete scripts for <command>buf</command>.
+      '';
+    };
```

Comment:
Aren't the completions picked up automatically? Looking at the `buf` package in Nixpkgs it does seem in generate and install the completions: https://github.com/NixOS/nixpkgs/blob/78bdd3bd437281896695e3947a81fa5609e9548c/pkgs/development/tools/buf/default.nix#L56-L60

Reply:
Ah yes, you are right. I've removed these options.